packages/mdc-fab/mdc-fab.scss

```diff
@@ -59,33 +62,6 @@
     @include mdc-ripple-fg((pseudo: "::after"));
   }
 
-  &::before {
-    position: absolute;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    transition: mdc-animation-exit(opacity, 120ms);
-    border-radius: inherit;
-    background: rgba(0, 0, 0, .2);
-    content: "";
-    opacity: 0;
-  }
-
-  &:focus::before {
```

Comment:
Since MDCRipple now provides styling for graceful degradation, we can remove the styles that used to handle it.